docs/tutorials/screen-resolution-detection.md

```diff
@@ -0,0 +1,38 @@
+# Screen resolution detection
+
+Shaka Player is able to detect the screen resolution and limit the resolution
+on certain devices:
+
+## Chromecast
+
+We can detect if the device supports 1920x1080 or 3840x2160 (4K), by default
```

Comment:
I would make the "by default" clause for each of these into a separate sentence.

"We can detect if ... .

By default, the maximum resolution ..."